docs/src/test-api/class-teststore.md

```diff
@@ -0,0 +1,59 @@
+# class: TestStore
+* since: v1.32
+* langs: js
+
+Playwright Test provides a global `store` object that can be used read/write values to the filesystem. Each value is stored in its own file with the key used as a path relative to `TestConfig.storeDir` (it is `./playwright` by default).
+
+```ts
+import { test, store } from '@playwright/test';
+
+test('get user name', async ({ page, context }) => {
+  await page.goto('/');
+  // Return mock user info from the store.
+  await page.route('**/info/user', route => route.fulfill({ path: store.path('mocks/user.json')}))
+  await page.getByText('My Profile');
+  // Check that the name matches mock data.
+  await expect(page.getByLabel('Name')).toHaveText('John');
+});
+```
+
+## async method: TestStore.get
+* since: v1.32
+- returns: <[any]>
+
+Get named item from the store. Returns undefined if there is no value with given path.
```

Comment:
We should figure out:
- whether we support binary values like images;
- whether we treat json specially and JSON.parse it;
- whether we detect mime type from extension or pass it explicitly.

Reply:
> whether we support binary values like images;
We should, I'll address this separately.


> whether we treat json specially and JSON.parse it;
I'd say yes, as this is expected to be the most common use case.

> whether we detect mime type from extension or pass it explicitly.
If the keys include extension like in this implementation to we should infer the mime type from it I believe. In any case I'll tackle this in a separate PR.